source/install/desktop-app-install.rst

```diff
@@ -128,19 +128,19 @@ Unofficial, community-driven ``.rpm`` packages are available.
 
 1. Download the latest version of the Mattermost Desktop App:
 
-- 32-bit systems
+  - 32-bit systems
   `mattermost-desktop-5.0.4-linux-i686.rpm <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-i686.rpm>`__
-- 64-bit systems
+  - 64-bit systems
   `mattermost-desktop-5.0.4-linux-x86_64.rpm <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-x86_64.rpm>`__
 
 2. At the command line, execute one of the following commands depending on the package that you downloaded:
 
-- 32-bit systems
+  - 32-bit systems
   ``sudo rpm -i mattermost-desktop-5.0.4-linux-i686.rpm``
-- 64-bit systems
+  - 64-bit systems
   ``sudo rpm -i mattermost-desktop-5.0.4-linux-x86_64.rpm``
 
-3. To run Mattermost, open **Dash** (located at the top left corner), enter **mattermost**, then click the Mattermost icon.
+3. To run Mattermost, open **Dash** (located at the top left corner), enter **mattermost**, then select the Mattermost icon.
```

Comment:
I wouldn't put concrete steps on how to run a linux app into this doc. We don't know what window manager the user is running so we can't assume that these are the correct steps to run the app.